doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,49 @@
+# GitHub discussions spec
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in the closed beta status. It will be released until the end of the year. 
+
+It's a new tab in the GitHub repo page where people can start new threads like in a forum.
+
+The threads must be added to a category. The answer feature can be enabled or disabled for categorys. If enabled a person with triage perm or the creator of the thread can mark a reply as answer.
+
+You can sort threads in the overview for categories and for (un-)answered.
+
+One thread can be pinned on the main page of the discussion section. (One with rules or something like that).
+
+Threads uses the same numeration like PR and issues. So they can be linked by the others. A conversation can also be locked like in issues and PR.
+
+## Why do we need this at PowerToys
```

Comment:
Why do we need this *for* PowerToys sounds way better

Reply:
ok.